tests/runtests/prepare.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+set -x
+
+dnf install -y \
```

Comment:
So this assumes that the tests will only be run for released versions only, right? What if we wanted to test nightlies, for instance?

Reply:
We would need a different TMT "plan", which in this case is `integration.fmf`.
That plan could prepare the environment differently.

I am pretty sure it would be possible to parametrize the current plan/script as well, I am just not that familiar with `tmt` yet :)

Reply:
Got it, thanks.

Reply:
If you have fresh rpm packages built locally installing them on the guest is easy:

    tmt run prepare --how install --directory path/to/rpms

This however overrides config from plan so you might want to add a commented section in the plan like we do [here](https://github.com/psss/tmt/blob/master/plans/main.fmf#L15) and uncomment when desired.

Reply:
Sweet, thanks.